backend.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/* Indiquez le chemin du dossier vers lequel vous avez stocké l'API Sendinblue */	
+require_once('./APIv3-php-library-master/autoload.php')
+/* Remplacez ici l'ID de liste par l'ID de la vôtre */	
+$list_id = "#2";
+/* Renseignez ici votre CLÉ API */	
+$API_key = "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";
```

Comment:
Is it smart to leave API key?

Reply:
Hi @kgplise @r0073rr0r It is not smart to leave api-key. 
and if somewhere you left your api-key by mistake or any reason, you can delete that key from you account, so nobody can use it, and for yourself you can create new one and can use them.